generic-functions.lisp

```diff
@@ -1,5 +1,10 @@
 (in-package #:ctype)
 
+(defmacro define-commutative-method (name arg1 arg2 &body body)
```

Comment:
define-commutative-method should look like defmethod - so the lambda list here should be `(name (arg1 arg2) &body body)`. This confused me a couple times looking through the code.

Reply:
Do you agree that it should still only be for methods taking 2 arguments?

Reply:
Yes. this is just a syntax thing